### DISCUSSION TITLE
Frontend Language + Framework

### DISCUSSION THREAD

**Discussion by umitcan07** [bounswe/bounswe2024group11#142] [Polls]
First, please fill out the [Google Form](https://forms.gle/8oeJ1dQ6iapBwmKS8):

**Comment by MucahitErdoganUnlu** [comment]
Is this discussion still relevant? @umitcan07

**Reply by Meminseeker** [reply]
We continue with React for the frontend. But I don't know whether it is with or without TypeScript; probably, the frontend team discusses and decides on it. When you do, please inform us, the frontend gang!

**Reply by umitcan07** [reply]
Yes, React with Typescript
